src/Components/DataHistoryExplorer/DataHistoryExplorer.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { ComponentStory } from '@storybook/react';
+import { getDefaultStoryDecorator } from '../../Models/Services/StoryUtilities';
+import DataHistoryExplorer from './DataHistoryExplorer';
+import { IDataHistoryExplorerProps } from './DataHistoryExplorer.types';
+import MockAdapter from '../../Adapters/MockAdapter';
+
+const wrapperStyle = { width: '100%', height: '500px', padding: 8 };
+
+export default {
+    title: 'Components/DataHistoryExplorer',
```

Comment:
Can we add a folder for DataHistoryExplorer and then put the modal and the component itself inside the folder?